src/drivers/gps/gps.cpp

```diff
@@ -795,6 +799,8 @@ GPS::task_main()
 				}
 			}
 
+#if !defined(__PX4_POSIX_RPI)
```

Comment:
This should be a command line param disabling auto-scan and hard-setting UBX as protocol.